### DISCUSSION TITLE
Disable Quasar Grid System

### DISCUSSION THREAD

**Discussion by mrmmg** [quasarframework/quasar#9957] [General - Components / Directives / etc] [answered]
Hi

I'm using quasar to create a Progressive Web Application (PWA). I'm want to use my own grid system on the quasar. how can I disable quasar grid styles?

**Comment by ontwikkelfabriek** [comment, marked as answer]
> my own grid system on the quasar

you don't want that. Quasar uses flexbox, if you use Quasar use flexbox.

**Reply by mrmmg** [reply]
i'm using bootstrap grid system and bs grid does not work with quasar grid system...!

**Reply by ontwikkelfabriek** [reply]
@mrmmg 

> i'm using bootstrap grid system and bs grid does not work with quasar grid system...!

That's correct, they will not work together. There are class/css/style conflicts between Quasar and Bootstrap Grid ( and tailwind , and ect....). So you can't use them together.( well you probably can but you really don't want to go into that rabbit hole).

Quasar uses Flexbox, if you want to create apps with Quasar (cli) you have to use and learn Flexbox.

**Comment by rhscjohn-dev** [comment]
You could try CSS Grid.  Grid and flex play nice together  most of the time.  But sometimes they get into a fight and you have to step in and straighten things out.
